verification/src/settings.rs

```diff
@@ -0,0 +1,173 @@
+use crate::consts::DEFAULT_COMPILER_LIST;
+use anyhow::anyhow;
+use config::{Config, File};
+use cron::Schedule;
+use serde::{de::IgnoredAny, Deserialize};
+use std::{net::SocketAddr, num::NonZeroUsize, path::PathBuf, str::FromStr};
+use url::Url;
+
+#[derive(Debug, Clone, Default, Deserialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct Settings {
+    pub server: ServerSettings,
+    pub solidity: SoliditySettings,
+    pub sourcify: SourcifySettings,
+    pub metrics: MetricsSettings,
+
+    #[serde(rename = "config")]
+    pub config_path: IgnoredAny,
```

Comment:
why do we even need this? it's kinda strange to have config path in the config, we don't use it anywhere, ...?

Reply:
I've added a comment describing why it is required 

The main idea is that we deny all unknown fields, so that users could notice any errors (otherwise, if a user make any mistake in the name of the field we will just use a default value without any warning or notification). At the same time, path to configuration file is setup through `PREFIX__CONFIG` (`VERIFICATION__CONFIG`, in our case) environment variable where the prefix is used to discriminate variables our service should consider. Thus, if we remove `config_path` field, `config` value would be considered as unknown during deserialization

Reply:
ah, it would fail during ENV deserialization?

Reply:
Yeap